borg/archiver.py

```diff
@@ -1213,6 +1216,8 @@ def build_parser(self, args=None, prog=None):
                                help='only check last N archives (Default: all)')
         subparser.add_argument('-P', '--prefix', dest='prefix', type=str,
                                help='only consider archive names starting with this prefix')
+        subparser.add_argument('--no-progress', dest='no_progress', action='store_false',
```

Comment:
I'ld prefer to not having --no-whatever options.

Or, said without double negation: I prefer --whatever options. :)